search_events_app/tests.py

```diff
@@ -9,50 +9,56 @@
 from search_events_app.models.feature import Feature
 
 
-class TestModels(TestCase):
+class TestCountry(TestCase):
+
+    def setUp(self):
+        self.country = Country.objects.create(name="Argentina", alpha_2_code="AR",
+        alpha_3_code="ARG", flag="https://restcountries.eu/data/arg.svg")
+
+    def test_country_basic_info(self):
+        self.assertEqual(self.country.name, "Argentina")
+        self.assertEqual(self.country.alpha_2_code, "AR")
+        self.assertEqual(self.country.alpha_3_code, "ARG")
+        self.assertEqual(self.country.flag, "https://restcountries.eu/data/arg.svg")
+
+    def test_country_str(self):
+        self.assertEqual(self.country.__str__(), "Argentina")
+
+    
+class TestEvent(TestCase):
+
     def setUp(self):
         self.country = Country.objects.create(name="Argentina", alpha_2_code="AR",
         alpha_3_code="ARG", flag="https://restcountries.eu/data/arg.svg")
+        self.event2 = Event("Name", "www.google.com")
```

Comment:
If you use this property only in _test_event_basic_info_ method, you should move it to the method... Because you`re creating this object for each method and it isn't necessary.